cmd/bootstrap/cmd/finalize.go

```diff
@@ -79,7 +79,7 @@ func addFinalizeCmdFlags() {
 	_ = finalizeCmd.MarkFlagRequired("partner-stakes")
 
 	// required parameters for generation of root block, root execution result and root block seal
-	finalizeCmd.Flags().StringVar(&flagRootChain, "root-chain", "emulator", "chain ID for the root block (can be \"main\", \"test\" or \"emulator\"")
```

Comment:
is it possible that we need to fix emulator codes to pass this flag?

Reply:
What do you mean by fix emulator codes?

Reply:
I mean is it possible a part of emulator code assumes this flag is always set to "emulator" and not pass it as a flag?

Reply:
The emulator doesn't actually use this tool. This tool was just using the `Emulator` chain ID for other, non-emulator chains rather than having a separate chain ID.